dags/common/enhancer.py

```diff
@@ -42,6 +43,18 @@ def __construct_titles(self, item, publisher):
             }
         ]
 
+    def __remove_country(self, item):
+        for author in item.get("authors", []):
+            for affiliation in author.get("affiliations", []):
+                pattern_for_cern_cooperation_agreement = re.compile(
+                    r"cooperation agreement with cern", re.IGNORECASE
```

Comment:
if we have the `compile` inside the loops it's loosing its advantage over the simple `match`, especially in a double for loop.

Reply:
@ErnestaP ☝️